hikari/impl/special_endpoints.py

```diff
@@ -1297,6 +1348,106 @@ def build(self) -> data_binding.JSONObject:
         return data
 
 
+@attr_extensions.with_copy
+@attr.define(kw_only=True, weakref_slot=False)
+class TextInputBuilder(special_endpoints.TextInputBuilder[_ContainerProtoT]):
```

Comment:
```suggestion
class ModalTextInputBuilder(special_endpoints.TextInputBuilder[_ContainerProtoT]):
```

Reply:
Unfortunately, I have decided to decline this suggestion.